apps/mobile/src/app/(home)/recover-wallet.tsx

```diff
@@ -78,10 +78,14 @@ export default function RecoverWallet() {
     }
   }
 
-  function onChangeText(text: string) {
+  async function onChangeText(text: string) {
     checkMnemonic(text);
     setRecoveryMnemonic(text);
     setInputState('default');
+    const copiedString = await Clipboard.getStringAsync();
```

Comment:
Logic here is that if they change the text and the change is the same as they clipboard contents then we should dismiss the keyboard